README.md

```diff
@@ -0,0 +1,109 @@
+</p>
+
+*The official Python client for Elasticsearch.*
+
+
+## Features
+
+* Translating basic Python data types to and from JSON
+* Configurable automatic discovery of cluster nodes
+* Persistent connections
+* Load balancing (with pluggable selection strategy) across available nodes
+* Failed connection penalization (time based - failed connections won't be
+  retried until a timeout is reached)
+* Support for TLS and HTTP authentication
+* Thread safety across requests
+* Pluggable architecture
+* Helper functions for idiomatically using APIs together
+
+
+## Installation
+
+[Download the latest version of Elasticsearch](https://www.elastic.co/downloads/elasticsearch)
+or
+[sign-up](https://cloud.elastic.co/registration?elektra=en-ess-sign-up-page)
+for a free trial of Elastic Cloud.
+
+Refer to the [Installation section](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_installation) 
+of the getting started documentation.
+
+
+## Connecting
+
+Refer to the [Connecting section](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_connecting)
+of the getting started documentation.
+
+
+## Usage
+-----
+
+* [Creating an index](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_creating_an_index)
+* [Indexing a document](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_indexing_documents)
+* [Getting documents](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_getting_documents)
+* [Searching documents](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_searching_documents)
+* [Updating documents](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_updating_documents)
+* [Deleting documents](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_deleting_documents)
+* [Deleting an index](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/getting-started-python.html#_deleting_an_index)
+
+
+## Compatibility
+
+Language clients are forward compatible; meaning that the clients support
+communicating with greater or equal minor versions of Elasticsearch without
+breaking. It does not mean that the clients automatically support new features
+of newer Elasticsearch versions; it is only possible after a release of a new
+client version. For example, a 8.12 client version won't automatically support
+the new features of the 8.13 version of Elasticsearch, the 8.13 client version
+is required for that. Elasticsearch language clients are only backwards
+compatible with default distributions and without guarantees made.
+
+| Elasticsearch Version | Elasticsearch-PHP Branch | Supported |
```

Comment:
```suggestion
| Elasticsearch Version | Elasticsearch-Python Branch | Supported |
```

Reply:
Oh wow, great catch! PHP already had the table in Markdown so I copied it and missed that it said PHP. Thanks for reading carefully.